TechnicalDocuments/CompomentComposition.md

```diff
@@ -0,0 +1,64 @@
+# Bento Component Composition Using `UIStackView`
+
+#### Updated at: 23-3-2019
+
+## The scope of this document 
+Component Composition with `Bento` is a very wide topic due to its flexible design.
+`Bento` components can be composed in many ways.
+
+The purpose of this document is to provide documentation for __composing Bento components using `UIStackView`__.
+Other ways of composition are out of the scope of this document.
+
+## How to use component composition in your framework
```

Comment:
Which framework?

Reply:
What do you mean?

It means how to use `Component Composition` in your application/framework/screen.

Reply:
>  I thought that if you need to implement a custom UI that can't use any of existing components - go on and compose atomic components in your renderer. The explanation though makes the emphasis on that it's not even possible. If that's really the case then what's the solution?

Atomic components are only available in `BabylonUI`.
New components can be added, but they have to be in `DesignLibrary`.
`Renderer`'s shouldn't compose components.